src/lib/nvme_pt_ioctl.c

```diff
@@ -1678,6 +1713,28 @@ int opal_activate_lsp_pt(struct sed_device *dev, const struct sed_key *key,
 	return ret;
 }
 
+int opal_revertsp_pt(struct sed_device *dev, const struct sed_key *key, bool keep_global_rn_key)
```

Comment:
suggest changing function name  to indicate that Locking SP is reverted

Reply:
done